libpod/image/image_test.go

```diff
@@ -185,3 +186,13 @@ func Test_splitString(t *testing.T) {
 	assert.Equal(t, splitString("a/foo/bar"), "bar")
 	assert.Equal(t, splitString("bar"), "bar")
 }
+
+// Test_stripSha256 tests test the stripSha256 function which removes
+// the prefix "sha256:" from a string if it is present
+func Test_stripSha256(t *testing.T) {
+	assert.Equal(t, stripSha256(""), "")
+	assert.Equal(t, stripSha256("test1"), "test1")
+	assert.Equal(t, stripSha256("sha256:9110ae7f579f35ee0c3938696f23fe0f5fbe641738ea52eb83c2df7e9995fa17"), "9110ae7f579f35ee0c3938696f23fe0f5fbe641738ea52eb83c2df7e9995fa17")
+	assert.Equal(t, stripSha256("sha256:9110ae7f"), "9110ae7f")
+	assert.Equal(t, stripSha256("sha256:"), "sha256:")
```

Comment:
Can you add a test for sha256 + 1 character? Like "sha256:a" turns into "a"?

Reply:
k